packages/client/src/comp/Footer.tsx

```diff
@@ -162,9 +163,9 @@ export const Footer = () => {
     <Container>
       <Logo>
         <img src={logo} alt="VoteByMail"/>
-        <Link to='/privacy-policy' onClick={scrollToTop}>
+        <a href={privacy_policy_url} onClick={scrollToTop}>
```

Comment:
Hey Luca, just a quick reminder that you can safely remove `onClick={scrollToTop}` and its declaration on the lines above.

This was added when the policy was displayed within the page, it was used to scroll to the top of the website when users clicked on the Privacy Policy--so they wouldn't have to do it themselves. I don't think it will be useful now, since the file is going to open externally.

Cheers!

Reply:
thanks!